javaagent-core/src/main/java/org/hypertrace/agent/core/instrumentation/utils/ContentTypeUtils.java

```diff
@@ -66,11 +65,24 @@ public static String parseCharset(String contentType) {
     return null;
   }
 
-  public static String convertToJSONString(Object obj) {
-    try {
-      return new ObjectMapper().writeValueAsString(obj);
-    } catch (JsonProcessingException e) {
-      throw new RuntimeException(e);
+  public static String convertToJSONString(Map<?, ?> map) {
```

Comment:
How does this change show request body for x-www-form-urlencoded HTTP requests?

Reply:
It's a utility method for converting string Maps to JSON.  In order for Span attributes to show correctly in the UI, they must be in JSON format.  However, as indicated in my other comment, I'll be changing the implementation to use Jackson to perform this conversion.  I'll also add some Javadoc for this method.